src/samples/WorkflowCore.SampleSqlServer/README.md

```diff
@@ -0,0 +1,25 @@
+﻿# SQL Server sample
```

Comment:
Not sure we need a whole sample project to show the usage of a particular provider